docs/modules/data-structures/pages/vector-search-tutorial.adoc

```diff
@@ -0,0 +1,183 @@
+    hazelcast.logging.type: log4j2
+    hazelcast.partition.count: 13
+
+  jet:
+    enabled: True
+    resource-upload-enabled: True
+
+  vector-collection:
+    images:
+      indexes:
+        - name: semantic-search
+          dimension: 512
+          metric: COSINE
+        
+
+```
++
+* `hazelcast.partition.count: Vector search performs better with fewer partitions. On the other hand, fewer partitions means larger partitions, which can cause problems during migration.  A discussion of the tradeoffs can be found here
+(https://docs.hazelcast.com/hazelcast/latest/data-structures/vector-search-overview#partition-count-impact).
+* `jet`:  This is the Hazelcast stream processing engine.  Hazelcast pipelines are a scalable way to rapidly ingest or process large amounts of data.  In this example, we use a pipeline to compute embeddings and load them into a vector collection, so stream processing must be enabled.
+* `vector-collection`: If you are using a vector collection, you must configure the index settings. There are no defaults. In this case, the name of the collection is `images` and it has one index, which is called `semantic-search`.  The dimension and distance metric are dependent on the embedding being used. The `dimension` must match the size of the vectors produced by the embedding.  The `metric` defines the algorithm used to compute the distance between 2 vectors and it must match the one used to train the embedding.  We are using the CLIP sentence transformer for embeddings. CLIP uses a dimension of 512 and cosine distance metric (literally the cosine of the angle between 2 vectors, adjusted to be non-negative). The options supported are documented here (https://docs.hazelcast.com/hazelcast/latest/data-structures/vector-collections).
+
+. Start the tutorial environment.
++
+```sh
+docker compose up -d
+```
++
+This launches Hazelcast Platform, Hazelcast Management Center, and the Web server. Hazelcast Management Center is accessible at http://localhost:8080.
+
+. Using your Java IDE, open `ImagesIngestPipeline.java` in the `image-ingest-pipeline` module. Follow the guidance and instructions in the file. 
+
+. Test the pipeline
++
+.. build the project: `mvn clean package`
+.. deploy the pipeline: `docker compose run deploy-image-loader`
+.. monitor the logs: `docker compose logs --follow hz`
+.. to generate events, copy some images from the `images` folder
+into the `www` folder.  You will see a new pipeline event for each image.
+.. To stop the job, use the Hazelcast Management Center. Navigate
+to "Jobs" and select "Stop".
++
+Iterate on the pipeline until you have finished the whole thing, and you are sinking vectors into Hazelcast with no errors.
++
+Once you have the python service in your pipeline, it will take some time to initialize (up to 5 minutes) because it has to download and install many Python packages.  You will see something like the following in the hazelcast logs when the Python stream stage has initialized.
```

Comment:
```suggestion
Once you have the python service in your pipeline, it will take some time to initialize (up to 5 minutes) because it has to download and install many Python packages. You will see something like the following in the hazelcast logs when the Python stream stage has initialized.
```
You might need to do a search for double spaces...going to stop highlighting them lol

Reply:
Will do.